### ISSUE TITLE
ImportError: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found

### ISSUE THREAD

**Issue by bazingagin** [bamler-lab/constriction#1]
There is an error when `import constriction` on Ubuntu system with python.

Environment:
Ubuntu 18.04
python 3.7, 3.8, 3.9

**Comment by timxzz** [comment]
+1

I run the following lines on Google Colab: 

```
!pip install --upgrade constriction bumpy
import constriction
```
, which results in the same error:

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-1570388c3690> in <module>()
      1 get_ipython().system('pip install --upgrade constriction bumpy')
----> 2 import constriction

ImportError: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /usr/local/lib/python3.7/dist-packages/constriction.cpython-37m-x86_64-linux-gnu.so)

---------------------------------------------------------------------------
```

**Comment by robamler** [comment]
Thank you for reporting and sorry for getting back to you this late. This should be fixed as of version 0.2.1, which now follows best practices when compiling python extension packages for linux.

Running the following on Google Colab now works:

```bash
!pip install constriction~=0.2.1  # (no need to explicitly mention numpy anymore it's a dependency now)
import constriction
```

Please feel free to reopen if you're still experiencing an issue.
